forc-test/src/execute.rs

```diff
@@ -23,10 +21,85 @@ use vm::interpreter::{InterpreterParams, MemoryInstance};
 use vm::state::DebugEval;
 use vm::state::ProgramState;
 
+#[derive(Debug, Clone)]
+pub enum CapturedEcal {
+    Write { fd: u64, bytes: Vec<u8> },
+    Unknown { ra: u64, rb: u64, rc: u64, rd: u64 },
+}
+
+impl CapturedEcal {
```

Comment:
maybe we can move this to another file?
i.e. `ecal.rs` or something?

Reply:
Done.